Lesson 15

MariaZasypkina opened the pull request:
Had to make separete commits here, on the web because could not do it through tesminal in vscode (working on solving this problem)

In this pull request the following changes were made:

- added photo to the 'about' section and added a mobile style for it
- fixed 'skills' section styles
- message form styles added 
- list of repos added with the fetch API
links to the Open API project are added in the readMe file

thank you!

MariaZasypkina committed:
Update index.css

MariaZasypkina committed:
Update index.js

MariaZasypkina committed:
Update README.md

MariaZasypkina committed:
Update index.html

kiramiller425 commented:
Hi Maria, 

For Assignment 15:

In the Portfolio, currently your code creates an **additional** UL in the Projects section. The instructions actually want the `projectList` variable to  "...query the `projectSection` (instead of the entire document) to select the (existing) ul element". Please change the code here:
```

  //Create a 'ul' list 

  let projectList = document.createElement('ul');
  
```
To **query** **instead** of create the ul.

Good job on the Open API project! Just make the changes to your portfolio project above and resubmit Assignment 15!

MariaZasypkina committed:
Update index.js

js code fixed in line 168 (replaced the creating new 'ul' with finding the existing one)

MariaZasypkina commented:
thank you, Kira!
I fixed the 'ul' issue.

kiramiller425 commented:
Great job! You're ready for the next lesson!
